doc/source/whatsnew/index.rst

```diff
@@ -10,13 +10,22 @@ This is the list of changes to pandas between each release. For full details,
 see the commit logs at http://github.com/pandas-dev/pandas. For install and
 upgrade instructions, see :ref:`install`.
 
+Version 1.1
+-----------
+
+.. toctree::
+   :maxdepth: 2
+
+   v1.1.0
+
 Version 1.0
 -----------
 
 .. toctree::
    :maxdepth: 2
 
    v1.0.0
+   v1.0.1
```

Comment:
this should also be removed or not critical?

Reply:
nvm.

Reply:
I think since I linked to this PR for the 1.1 whatsnew, This PR should retain the 1.1 setup and open another PR for 1.0.1 (but only after 1.0 is released)

Reply:
This PR is for 1.0.1